source/documentation/deploying_services/postgresql.md.erb

```diff
@@ -345,65 +345,71 @@ GOV.UK PaaS supports the following optional extensions, but they are not enabled
 
 |Extension|PostgreSQL version|
```

Comment:
I'm starting to wonder if we couldn't make this table better

Extension | 10 | 11 | 12 | 13 
-- | -- | -- | -- | -- | -- | -- | --
address_standardizer|✓|✓|✓|✓
chkpass|✓|✓|✓|𐄂

(I can't make Github's markdown parser happy with that table so sod it, syntax only)

Reply:
Ug fighting with table syntaxes. A little digging suggests our markdown parser is `kramdown`? https://kramdown.gettalong.org/syntax.html#tables